src/lib/pages/backoffice/Actions/CheckIn/ItemsSearch/ItemsSearch.js

```diff
@@ -20,6 +25,7 @@ export default class ItemsSearch extends Component {
         <SearchBarILS
           onSearchHandler={this.executeCheckinAndClearInput}
           placeholder="Scan physical copy barcode to check-in..."
+          ref={this.searchBarRef}
```

Comment:
Ideally, this is not the recommended way in React. Ideally, the outer cmp should have a state that is injected in the inner cmps. Outer components should not know internal implementations of other cmps. If it is not exposed via a prop, then it is a private method.

Having said so, I understand it is tricky to refactor it now.
Please keep this in mind because it should not become a pattern and my fear is that we copy/paste this and it becomes an habit.

Reply:
that is true, it is not the best way to deal with this problem, but I have chosen it because of the alternative being tricky to refactor, especially for such a small given value